ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/StreamSelectKeyBuilderTest.java

```diff
@@ -148,7 +157,7 @@ public void shouldPassCorrectArgsToParamBuilder() {
     // Then:
     verify(paramBuilder).build(
         SOURCE_SCHEMA,
-        KEY,
+        stream.getKeySerdeFactory(), KEY,
```

Comment:
nit: spacing

Reply:
:) auto-refactoring - thanks for catching